Docs/Design/2026-02-25-acp-centric-workflow-pipeline-design.md

```
@@ -0,0 +1,243 @@
+# ACP-Centric Workflow Pipeline Design (API-First)
+
+Date: 2026-02-25
```

markdownlint-cli2 (0.21.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)